architecture-decisions/0004-build-on-spotlight.md

```diff
@@ -0,0 +1,46 @@
+# 4. Build on Spotlight
+
+Date: Archaeological
+
+## Status
+
+Accepted
+
+## Context
+
+PUDL has [collection landing
+pages](http://pudl.princeton.edu/collections/pudl0058) because it's a digital collections site which
+staff pushed content into. PUDL required all items be in a single collection,
+and couldn't provide good search-across. Figgy is a staff back-end, so we need some sort of
+application to provide that functionality.
+
+Curators also had a history of either requesting or creating ad-hoc websites to showcase
+their material or accompany on-site exhibits. We wanted instead to provide a CMS for
+them to create those experiences based on material they curate which wouldn't
+create metadata silos and increased maintenance.
+
+Further, curators often had disagreements about metadata display with the
+cataloguing department. It was important that certain fields display a certain
+way, but be catalogued according to best practices.
```

Comment:
I think one other thing to include here is that it's sometimes important to display the metadata differently because it's in the context of an exhibit — e.g., you might have a few items with the same title where it makes sense to have the title be the same in the catalog/EAD, but you want to differentiate them in the exhibit.

Reply:
That seems like a better way to characterize this. I modified the paragraph.